code/modules/mob/mob.dm

```diff
@@ -969,9 +969,9 @@
 	var/turf/T = get_turf(src)
 	if(M.loc != T)
 		var/old_density = density
-		density = FALSE
+		density = FALSE // Hacky and doesn't use set_density()
```

Comment:
Thoughts on replacing manual changes like this with a quick_set_density(FALSE) macro? I'm not terribly thrilled about copy+pasting this comment wherever needed

Reply:
(Unless we plan on removing the need for this entirely in the near future)

Reply:
The long-term idea is to remove this if possible, and not expand on it.
It happens in two places in the code only, both commented.